CI tests are failing

**Describe the bug**
A few tests which were passing previously are now failing due to an change in the version of `lm_eval`.
```
=========================== short test summary info ============================
FAILED test/test_api.py::Test_WhittleLM::test_evaluate - ValueError: The repository for EleutherAI/logiqa contains custom code which must be executed to correctly load the dataset. You can inspect the repository content at https://hf.co/datasets/EleutherAI/logiqa.
Please pass the argument `trust_remote_code=True` to allow custom code to be run.
FAILED test/test_api.py::Test_WhittleLM::test_compare_litgpt - ValueError: The repository for EleutherAI/logiqa contains custom code which must be executed to correctly load the dataset. You can inspect the repository content at https://hf.co/datasets/EleutherAI/logiqa.
Please pass the argument `trust_remote_code=True` to allow custom code to be run.
FAILED test/test_evaluate.py::test_evaluate[litgpt-True-True] - ValueError: Loading this dataset requires you to execute custom code contained in the dataset repository on your local machine. Please set the option `trust_remote_code=True` to permit loading of this dataset.
====== 3 failed, 265 passed, 31 skipped, 39 warnings in 236.41s (0:03:56) ======
```

**To Reproduce**
Steps to reproduce the behavior:
Run the `unit-test` workflow on Github

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -22,6 +22,7 @@ dependencies = [
     "torch>=2.5",
     "transformers>=4.51.3,<4.54",
     "litgpt[extra]==0.5.10",
+    "lm-eval<0.4.9.1",
     "syne-tune[moo]==0.13.0",
     "torchvision>=0.18",
     "boto3==1.34.147",

